src/main/java/com/vinylteam/vinyl/security/SecurityConfiguration.java

```diff
@@ -50,14 +53,26 @@ public void configure(WebSecurity web) throws Exception {
     protected void configure(HttpSecurity http) throws Exception {
         http
                 .csrf().disable()
+                .cors(withDefaults())
                 //.sessionManagement()
                 //.sessionCreationPolicy(SessionCreationPolicy.STATELESS)
                 .authorizeRequests()
                 .anyRequest().permitAll()
                 .and()
-                .addFilterAfter(userAttributeFilter, UsernamePasswordAuthenticationFilter.class)
                 .addFilterBefore(jwtValidatorField, UsernamePasswordAuthenticationFilter.class)
-                .formLogin().loginPage("/signIn").permitAll().usernameParameter("email").defaultSuccessUrl("/profile");/*.successHandler()*///;
+                .formLogin().loginPage("/signIn").permitAll().usernameParameter("email").defaultSuccessUrl("/profile")
+                .and()
+                .logout().addLogoutHandler(logoutHandlerService).logoutSuccessUrl("/successlogout");
+    }
+
+    @Bean
+    public CorsConfigurationSource corsConfigurationSource() {
+        CorsConfiguration configuration = new CorsConfiguration();
+        configuration.setAllowedOrigins(Arrays.asList("http://localhost:3000", "http://react-wax-deals.herokuapp.com"));
```

Comment:
Please, change it 'http://react-wax-deals.herokuapp.com' to 'https://react-wax-deals.herokuapp.com'. I mean that request will be come from 'https'